rclpy/executors/examples_rclpy_executors/publish_pointcloud.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
```

Comment:
No need for the shebang, since this is going to be a console script.

Reply:
```suggestion
```